spec/unit/pdk/cli/test/unit_spec.rb

```diff
@@ -29,17 +29,53 @@
           expect(PDK::Test::Unit).to receive(:list).with(no_args).once.and_return([])
         end
 
-        it { expect { test_unit_cmd.run_this(args) }.to output(%r{No examples found}m).to_stdout }
+        it { expect { test_unit_cmd.run_this(args) }.to output(%r{No test files with examples found}m).to_stdout }
```

Comment:
What does the 'm' do in these expects?

Reply:
The `m` allows the regex to be matched across multiple lines.